### DISCUSSION TITLE
Make find highlight thing found

### DISCUSSION THREAD

**Discussion by jms19** [sumatrapdfreader/sumatrapdf#2537] [Ideas]
I spend a lot of time looking at schematics and trying to work out where signals go.

Often I'll search for a signal name, be taken to a page but simply cannot see the thing found. One workaround is to be zoomed in but it would be great if I could stay on a whole-page view and the thing I was looking for jumped out using colour, animation, a circle, some lines, larger text or some combination.

**Comment by GitHubRulesOK** [comment]
It is highlighted and you can change the translucent color but it needs to be translucent, many users would not like it flashing so like other apps it is steady, I appreciate on very large pages it is difficult to spot a small target thus as you say it is easiest done using zoom in, and SumatraPDF will zoom in towards that location. 
![image](https://user-images.githubusercontent.com/12754913/167107978-0f6bf774-6d22-41d1-b0fb-66e34a8f116e.png)

**Reply by gnuarm** [reply]
How do you change the color of the highlight?  I'm having the same problem, with the same thing, schematics.  Only this schematic has yellow highlights on most of the text as part of a checking scheme.  This makes it nearly impossible to find the search highlight.  

I see there is only a single control on the search itself, caps sensitive or ignore.  Perhaps another control could be added to zoom in, then back out, or indicate the found text with an arrow until the control is released?  The OP can zoom in easily enough, but then it's a bother to have to return to your previous zoom.  The momentary arrow could even be added as an automatic, momentary indicator. 

This is a significant issue, even if not too often.  Sometimes I get search hits on text that can't be seen.  If the arrow pointed to that, it would easily tell me there's nothing to look for.  

I just realized that the search in my browser does some of this.  Maybe I'm using the wrong PDF viewer?

**Reply by GitHubRulesOK** [reply]
@gnuarm 
color setting is `SelectionColor = #ff0000` but as stated above is semi-translucent
once an item is found zoom + button will move towards the selection so it appears larger and although zoom pivot origin is top left selection is kept fairly window relative
